pkg/snapshot/filesystem.go

```diff
@@ -276,17 +276,22 @@ func ensureFileSystemMinioDeployment(ctx context.Context, clientset kubernetes.I
 }
 
 func fileSystemMinioDeploymentResource(clientset kubernetes.Interface, secretChecksum string, deployOptions FileSystemDeployOptions, registryOptions kotsadmtypes.KotsadmOptions) (*appsv1.Deployment, error) {
```

Comment:
Do we still want kots to deploy minio pods in the upcoming releases?

Reply:
yes, to continue to support the people already using it